_source/no-search/p8s-remote-write.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: article
+title: Configuring remote write for Prometheus
+permalink: /user-guide/infrastructure-monitoring/p8s-remote-write.html
+flags:
+  logzio-plan: community
+  Beta: yes
+tags:
+  - metrics integrations
+contributors:
+  - yberlinger
+---
+Manage your metrics with Logz.io open source Infrastructure Monitoring, powered by Prometheus and Grafana.  Using a hosted solution lets you and your team focus on the bigger picture, with a handy roster of dashboards to oversee continuous deployment, CI/CD pipelines, and prevent outages, manage incidents, and remediate crashes in multi-microservice environments, hybrid infrastructures, and complex tech stacks.
+
+Integrate your Prometheus toolkit with the Logz.io Metrics platform to continuously monitor your system’s health and help you maintain system performance, speed, and resilience, and address problematic trends when they first appear - and before they become critical and threaten crashes or system outages.
+
+![Showcased Metrics dashboard](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/metrics-intro.png)
+
+To send your Prometheus application metrics to a Logz.io Infrastructure Monitoring account, use remote write to connect to Logz.io as the endpoint. Your data is formatted as JSON documents by the Logz.io listener. 
+
+If you have multiple Prometheus server instances, you'll have to add Logz.io as an endpoint for each instance. 
+
+By default, all metrics in your Prometheus server(s) will be sent to Logz.io. To drop or send specific metrics, add Prometheus labeling before enabling the remote write, or as part of the remote write configuration. You can read more about Prometheus relabeling tricks <a href ="https://medium.com/quiq-blog/prometheus-relabeling-tricks-6ae62c56cbda" target="_blank">here. <i class="fas fa-external-link-alt"></i> </a> 
```

Comment:
is it ok to use this guy blog as a reliable source from our docs?

Reply:
These are the links that Doron shared. @Doron-Bargo  - what do you say?